sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -704,8 +704,8 @@ object SQLConf {
   val PARQUET_FILTER_PUSHDOWN_INFILTERTHRESHOLD =
     buildConf("spark.sql.parquet.pushdown.inFilterThreshold")
       .doc("The maximum number of values to filter push-down optimization for IN predicate. " +
-        "Large threshold won't necessarily provide much better performance. " +
-        "The experiment argued that 300 is the limit threshold. " +
+        "Spark will push-down a value greater than or equal to its minimum value and " +
```

Comment:
I think the default value `10` is too smaller here.  What is the default threshold in IMPLA?

Reply:
Impala only optimize it to `>= minimum value` and `<= maximum value`: https://github.com/apache/impala/commit/aa05c6493b0ff8bbf422a4c38cf780bde34d51c7